docker/dev/cassandra-config.yml

```diff
@@ -0,0 +1,1302 @@
+
```

Comment:
Is it possible to not to use this file? I think you only want to change some of the config opts so is that possible to override via other way like env?

Reply:
No, I had a look. You can't make this Cassandra docker image (https://hub.docker.com/_/cassandra) work with Auth without updating the yaml.

Its not strictly necessary if we are comfortable not testing the auth path of the gocql client.

Reply:
yeah I was about to say the same thing. Let's skip the test for this. 
Cadence doesn't have a convention to test every config like those. It's okay not to test because if they don't work, we will know :P

Reply:
sure, i'll make the updates